packages/block/src/blockItemStore.ts

```diff
@@ -107,11 +111,19 @@ export class BlockItemStore<TBlock extends IBlockStub> extends StartStopService
     };
 
     // Type safe methods to store the anchor state for each block, indexed by component (used in the BlockchainMachine)
+    private mIsAnchorStateInitialised = false;
+    /**
+     * True when no at least one anchor state state was saved into the store, false otherwise.
+     */
+    public get isAnchorStateInitialised() {
+        return this.mIsAnchorStateInitialised;
```

Comment:
Maybe a bit more clear call this "hasAnyAnchorStates" - since it's on the blockitemstore.

Reply:
Changed in 31aeed1.